feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/widget/SessionToolbarBindingAdapter.kt

```diff
@@ -0,0 +1,11 @@
+package io.github.droidkaigi.confsched2019.session.ui.widget
+
+import android.widget.LinearLayout
+import androidx.coordinatorlayout.widget.CoordinatorLayout
+import androidx.databinding.BindingAdapter
+
+@BindingAdapter("setToolbarTitle")
+fun LinearLayout.setToolbarTitle(title: String?) {
+    (this.layoutParams as CoordinatorLayout.LayoutParams).behavior =
+        SessionToolbarBehavior(context, null, title ?: "")
```

Comment:
How about setting in Fragment?
This is very small and It is used only once.

Reply:
And you can use non null variables that is not need to null checks 👀

Reply:
I replaced it to SessionDetailFragment.